test/k8sT/Policies.go

```diff
@@ -640,6 +614,112 @@ var _ = Describe("K8sPolicyTest", func() {
 				validateConnectivity(WorldConnectivityDeny, ClusterConnectivityDeny)
 			})
 		})
+
+		Context("Validate CNP update", func() {
+			const (
+				allowAll     = true
+				denyFromApp3 = false
+			)
+
+			var (
+				cnpUpdateAllow        = helpers.ManifestGet("cnp-update-allow-all.yaml")
+				cnpUpdateDeny         = helpers.ManifestGet("cnp-update-deny-ingress.yaml")
+				cnpUpdateNoSpecs      = helpers.ManifestGet("cnp-update-no-specs.yaml")
+				cnpUpdateDenyLabelled = helpers.ManifestGet("cnp-update-deny-ingress-labelled.yaml")
+			)
+
+			validateL3L4 := func(allowApp3 bool) {
+				res := kubectl.ExecPodCmd(
+					helpers.DefaultNamespace, appPods[helpers.App2],
+					helpers.CurlFail(fmt.Sprintf("http://%s/public", clusterIP)))
+				ExpectWithOffset(1, res).Should(helpers.CMDSuccess(),
```

Comment:
fmt.Sprintf is not needed here

Reply:
Where do I keep copy/pasting this from, I always make this mistake ;)

Reply:
Not all are updated TBH, only the new ones :-D